src/mantarray_waveform_analysis/transforms.py

```diff
@@ -217,7 +218,38 @@ def calculate_displacement_from_voltage(
     Returns:
         A 2D array of time vs Displacement
```

Comment:
it might be good here to note here in the docstring that the units of "displacement" are meters

Reply:
I would also add a comment about the date those conversion values were obtained (today), and who they were obtained from (Kevin Grey), so we know if we look back in the future